demo/EuroBioC2023/workflow.Rmd

```diff
@@ -0,0 +1,437 @@
+- The research is based on sequencing (characterization of genes and species).
+- Nowadays, multiomics approach is more common (integration of taxonomy information with metabolite data, for example)
+- Computational methods are the new microscope
+- The research has expanded rapidly in previous years
+
+```{r pubmed_fig, fig.width=6, fig.cap="PubMed publications per year with a search term 'microbiome' (fetched: Sep 5, 2023)", class.source = 'fold-hide'}
+# Plot publication graph
+path <- "data/PubMed_Timeline_Results_by_Year.csv"
+df <- read.csv(path, skip = 1)
+
+x <- "Year"
+y <- "Count"
+
+plot <- ggplot(df, aes(x = .data[[x]], y = .data[[y]])) +
+    geom_bar(stat="identity")
+plot
+```
+
+### Big data
+
+- Most of the cohort studies require multiple types of data, for instance microbiome, phenotype and other omics data.
+- Linking these types of data could present multiple challenges; to mention:
+  * Different sources the data is coming form, could use different types of key identifier of the samples or subjects of the study; which could poses a problem in some cases for linking the data together.
+  * The choice of the data structure to combine and store these mulitple types of data is critical as well, in terms of the management, handling and wrangling of the data; to be able to perform the analysis.
+- Cohort datasets are large in size, which mostly one cannot conduct an analysis using a normal computer with limited hardware; HPC and cloud computing services are required in this case. 
+- Last but not least, a quality check reporting of the constructed object is substantially important to verify the correctness of the building process.   
+
+Regarding the choice of the data strcuture to handle cohorts with muliple data types or sources [MultiAssayExperiment](https://bioconductor.org/packages/release/bioc/html/MultiAssayExperiment.html),
+is considered to be very liable in such situation, easing-up the management and wrangling of the data; once is construced correctly. Moreover, several R packages frameworks are increasingly integrating 
+[MultiAssayExperiment](https://bioconductor.org/packages/release/bioc/html/MultiAssayExperiment.html) and the [SummarizedExperiment](https://bioconductor.org/packages/release/bioc/html/SummarizedExperiment.html)class, to provide users with a reliable and ease of use data structure.
+
+
+- Cohort datasets are large in size
+    - Data management, handling and wrangling --> data structure
+    - Computational power --> High performance computing (HPC) and cloud computing
+
+- [MultiAssayExperiment (MAE)](https://bioconductor.org/packages/release/bioc/html/MultiAssayExperiment.html) and [SummarizedExperiment (SE)](https://bioconductor.org/packages/release/bioc/html/SummarizedExperiment.html)
+    - Several R packages frameworks are increasingly integrating MAE and SE
+    - MAE enables linking of multiple experiments
+    - SE is an efficient data container to store data from an experiment
+
```

Comment:
I tied to shorten this little bit. Does it still have the most important info?

Reply:
should we highlight that TreeSE has many benefits? (it is SCE, and has metadata slot, feature sequence slot, altExps, trees etc. that SE doesnt have)